modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -458,13 +495,13 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
 
   private def renderGetMessage(field: Field) = field match {
     case field if field.tpe.isResolved && field.required =>
-      line"override def getMessage(): String = ${field.name}"
+      s"override def getMessage(): String = ${field.name}"
```

Comment:
why the change of `line` => `s` here ?

Reply:
Its probably a mistake. Ill change it back